sql/core/benchmarks/DateTimeBenchmark-results.txt

```diff
@@ -420,10 +420,12 @@ Conversion from/to external types
 
 OpenJDK 64-Bit Server VM 1.8.0_242-8u242-b08-0ubuntu3~18.04-b08 on Linux 4.15.0-1063-aws
 Intel(R) Xeon(R) CPU E5-2670 v2 @ 2.50GHz
-To/from java.sql.Timestamp:               Best Time(ms)   Avg Time(ms)   Stdev(ms)    Rate(M/s)   Per Row(ns)   Relative
+To/from Java's date-time:                 Best Time(ms)   Avg Time(ms)   Stdev(ms)    Rate(M/s)   Per Row(ns)   Relative
 ------------------------------------------------------------------------------------------------------------------------
-From java.sql.Timestamp                             394            399           5         12.7          78.7       1.0X
-Collect longs                                      1336           2676        1201          3.7         267.2       0.3X
-Collect timestamps                                 2025           2091          65          2.5         405.0       0.2X
+From java.sql.Date                                  935            947          10          5.3         187.1       1.0X
+Collect dates                                      2427           3239        1338          2.1         485.3       0.4X
```

Comment:
(485.3 - 187.1) = 298.2 ns/row after the changes
(461.1 - 111.8) = 349.3 ns/row on Spark 2.4.6-SNAPSHOT
/cc @cloud-fan @HyukjinKwon